code/modules/paperwork/paper.dm

```diff
@@ -298,6 +311,46 @@
 
 		user << "<span class='notice'>You stamp the paper with your rubber stamp.</span>"
 
+
+	var/list/book_burners = list(/obj/item/weapon/weldingtool, /obj/item/weapon/lighter, /obj/item/weapon/lighter)

```

Comment:
two of the same typepath in here, no mention of matches despite using them further down.